src/helpers/Math.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include <wayland-client.h>
+
+#include <hyprutils/math/Box.hpp>
+#include <hyprutils/math/Vector2D.hpp>
+#include <hyprutils/math/Mat3x3.hpp>
+
+using namespace Hyprutils::Math;
+
+eTransform          wlTransformToHyprutils(wl_output_transform t);
+wl_output_transform invertTransform(wl_output_transform tr);
```

Comment:
Same as in hyprland. I think we don't want to move `wlTransformToHyprutils` and `invertTransform` to hyprutils, as that would mean we have to get the `WL_OUTPUT_TRANSFORM*` definitions from somewhere right?

Also `Math.hpp` is included for Box, Vector2D and Mat3x3. But clangd complains that the header is not used directly. Let me know if we want the includes to work differently.

Reply:
looks fine